idx/idx_gen.go

```diff
@@ -1,6 +1,8 @@
 package idx
 
-// Code generated by github.com/tinylib/msgp DO NOT EDIT.
+// NOTE: THIS FILE WAS PRODUCED BY THE
+// MSGP CODE GENERATION TOOL (github.com/tinylib/msgp)
+// DO NOT EDIT
```

Comment:
looks like you're using an older version of the `msgp` tool.
please adjust the commit that introduces this change to only change what's necessary
(`git commit --fixup=<hash>; git rebase -i`)